document/CoreFormat.mdk

```diff
@@ -2112,6 +2109,16 @@ if any of those strings accurately describes the construct identified by this ob
 
 If none of those strings accurately describes the construct, `kind` may contain any value specified by the analysis tool.
 
+### `parentKey` property { #logicalLocation-parentKey }
+
+If the logical location represented by the `logicalLocation` object is a nested logical location,
+then the the `logicalLocation` object shall contain a property named `parentKey` whose value is a string 
+that matches the property name of the parent logical locations's `logicalLocation` object within
+`run.logicalLocations` (&sect;[#run-logicalLocations]).
```

Comment:
Yours is good. Fixed.

---
In reply to: [64121288](https://github.com/sarif-standard/sarif-spec/pull/213#discussion_r64121288) [](ancestors = 64121288)